src/patternfly/layouts/Level/level.hbs

```diff
@@ -1,3 +1,3 @@
-<div class="pf-l-level{{#if modifierClass }} {{ modifierClass }}{{/if}} Documentation-container-tall">
+<div class="pf-l-level{{#if level--modifier}} {{level--modifier}}{{/if}} Documentation-container-tall">
```

Comment:
You don't have to do this in this PR, but we should remove `Documentation-container-tall` and use `minHeight` in `index.js` so we don't have workspace classes in the actual layout code.

Reply:
I think this is addressed in this issue - https://github.com/patternfly/patternfly-next/issues/701

Reply:
ah! yep, awesome :)